components/src/core/drawer/drawer-header/drawer-header.component.scss

```diff
@@ -0,0 +1,53 @@
+.pxb-drawer-header-toolbar {
```

Comment:
Root class for each component has been using the pattern of `pxb-[component-name]`, please update.

Reply:
updated classname